Sources/Plasma/PubUtilLib/plPhysX/plPXSimulation.cpp

```diff
@@ -538,8 +538,12 @@ physx::PxRigidActor* plPXSimulation::CreateRigidActor(const physx::PxGeometry& g
         actor = fPxPhysics->createRigidStatic(globalPose);
         break;
     case plPXActorType::kDynamicActor:
-        actor = fPxPhysics->createRigidDynamic(globalPose);
-        break;
+        {
+            physx::PxRigidDynamic* dynamic = fPxPhysics->createRigidDynamic(globalPose);
+            dynamic->setMaxDepenetrationVelocity(10.f);
```

Comment:
What are the units used here?  The commit message says 20ft/s, but the value is 10.0.

Reply:
The commit message needs to say 10ft/s. 😞

Reply:
@Deledrius, I have corrected the commit message.